### PR TITLE
add elixir bridge to Bridge Troll

### PR DESCRIPTION
**Pull request by aneyzberg**
@lilliealbert @tjgrathwell I looked at the PR for Rust Bridge and Elm Bridge to figure out what needed to be done to add Elixir Bridge. I think this is it. But please let me know if I am missing something.  Thank you!

### CODE REVIEW

File: app/models/course.rb [hunk 1]
```diff
@@ -548,7 +548,54 @@ class Course < ActiveHash::Base
                 ]
             }
         ]
-    }
+    }, {
+      id: 15,
+      name: 'ELIXIR',
+      title: 'Elixir and  Phoenix',
```
Comment (lilliealbert) [inline app/models/course.rb]: Can you remove the extra space here?

**Comment by aneyzberg** [comment]
@lilliealbert fixed. sorry about that. I saw 'RUST' was in all caps...so was following that. clearly not the right way to go though.

**Comment by tjgrathwell** [comment]
:+1: it is deployed :rocket:

**Comment by aneyzberg** [comment]
@tjgrathwell thank you!!

**Pull request merged**